beacon-chain/powchain/service.go

```diff
@@ -424,15 +421,6 @@ func (s *Service) dialETH1Nodes(endpoint network.Endpoint) (*ethclient.Client, *
 		httpRPCClient.Close()
 		httpClient.Close()
 	}
-	syncProg, err := httpClient.SyncProgress(s.ctx)
-	if err != nil {
-		closeClients()
-		return nil, nil, err
-	}
-	if syncProg != nil {
-		closeClients()
-		return nil, nil, errors.New("eth1 node has not finished syncing yet")
-	}
```

Comment:
We would avoid connecting to geth if it is syncing. However, this behavior needs to change for the merge. We must connect to geth even if it is syncing